src/autocomplete.ts

```diff
@@ -101,6 +151,15 @@ export default class Autocomplete {
     }
   }
 
+  updateFeedbackForScreenReaders(inputString: string): void {
+    setTimeout(() => {
+      if (this.feedback) {
+        this.feedback.innerHTML = inputString
+        this.container.dispatchEvent(new CustomEvent('sr-update'))
+      }
+    }, SCREEN_READER_DELAY)
```

Comment:
So Typescript was very unhappy with me (first screenshot), so to avoid using two `if (feedback)`s, I opted for this. However, it's a tradeoff with the timeout.
![if outside of setTimeout; TS shows error](https://user-images.githubusercontent.com/14206003/141326508-20383533-af9c-414d-81de-124c474c3cb7.png)

This make TS happy, but two `if`s:
![two if feedbacks, but TS has no error](https://user-images.githubusercontent.com/14206003/141326706-b7129392-874b-4b5e-b2f8-524f7dc32676.png)

Reply:
WDYT? @koddsson

Reply:
Oh, that makes sense. What you had is the correct thing then.